src/components/AsideHeader/types.tsx

```diff
@@ -37,6 +37,9 @@ export interface AsideHeaderGeneralProps extends QAProps {
     }) => React.ReactNode;
     onClosePanel?: () => void;
     onChangeCompact?: (compact: boolean) => void;
+    onOpenEditMode?: () => void;
```

Comment:
I suggest moving the new callbacks under the `EditMenuOptions` prop, since they relate to menu editing, and at the top level of the AsideHeader component props it is difficult to guess by their names what they are for